aspnetcore/host-and-deploy/blazor/webassembly.md

```diff
@@ -23,13 +23,43 @@ The following deployment strategies are supported:
 * The Blazor app is served by an ASP.NET Core app. This strategy is covered in the [Hosted deployment with ASP.NET Core](#hosted-deployment-with-aspnet-core) section.
 * The Blazor app is placed on a static hosting web server or service, where .NET isn't used to serve the Blazor app. This strategy is covered in the [Standalone deployment](#standalone-deployment) section, which includes information on hosting a Blazor WebAssembly app as an IIS sub-app.
 
-## Precompression
+## Compression
 
-When a Blazor WebAssembly app is published, the output is precompressed to reduce the app's size and remove the need for runtime compression. The following compression algorithms are used:
+When a Blazor WebAssembly app is published, the output is statically compressed during publish to reduce the app's size and remove the overhead for runtime compression. The following compression algorithms are used:
 
 * [Brotli](https://tools.ietf.org/html/rfc7932) (highest level)
 * [Gzip](https://tools.ietf.org/html/rfc1952)
 
+Blazor's relies on the host to the server the appropriate compressed file. When using an ASP.NET Core hosted project, the host project is capable of performing content negotiation and serving the statically compressed file. When hosting a Blazor WebAssembly self-hosted application, you may have to do additional work to ensure statically compressed are served.
+
+* For IIS *web.config* compression configuration, see the [IIS: Brotli and Gzip compression](#brotli-and-gzip-compression) section. 
+* When hosting on static hosting solutions (such as GitHub Pages) that do not support this form of content negotiation, consider configuring your application to fetch and decode Brotli compressed files:
```

Comment:
It might be useful to point to this from the perf docs

Reply:
Do you want to add it over there to this PR, or do you want me to do it later?

The cross-link markdown is ...

```
<xref:host-and-deploy/blazor/webassembly#compression>
```

Reply:
I can do it in a follow up.

Reply:
Ok, I'll hold off on the merge.

Reply:
> follow up

Follow-up *commit* or follow-up *PR*?